### DISCUSSION TITLE
request etcd endpoint 'http://etcd:2379/version' error, temporary failure in name resolution

### DISCUSSION THREAD

**Discussion by ztf219** [apache/apisix#4951] [Q&A]
#将 Apache APISIX 的 Docker 镜像下载到本地
git clone https://github.com/apache/apisix-docker.git
# 将当前的目录切换到 apisix-docker/example 路径下
cd apisix-docker/example
# 运行 docker-compose 命令，安装 Apache APISIX
docker-compose -p docker-apisix up -d

执行完上面操作后，apisix一直在重启，报错：
request etcd endpoint 'http://etcd:2379/version' error, temporary failure in name resolution

**Comment by tokers** [comment]
Could you try to login to the apisix container and resolve the `etcd` manually.

**Reply by ztf219** [reply]
Yes, but I prefer that the image itself is no problem

**Reply by tokers** [reply]
What I want to know is the result of DNS resolution inside the apisix container, by using utilities like dig or nslookup.
